tasks/main.yml

```diff
@@ -14,7 +14,34 @@
     # it.
     state: latest  # noqa 403
     update_cache: yes
-  when: ansible_os_family == "Debian"
+  when: >
+    ansible_os_family == "Debian" and
+    ansible_distribution_release != "kali-rolling"
+
+- name: Reinstall cryptsetup-initramfs so upgrade doesn't fail
+  apt:
+    name: 'cryptsetup-initramfs'
+    state: latest  # noqa 403
+    update_cache: yes
+  when: ansible_distribution_release == "kali-rolling"
+
+- name: apt-get update
+  shell: apt-get update  # noqa 303 305
```

Comment:
Please add a brief comment explaining why you are skipping checks 303 and 305 here.  Same goes for line 38 below.

Reply:
Addressed in commit 9f51229

Reply:
You don't need `shell` here since you aren't using any shell features (like file redirection).  `command` will work, and should let you get rid of at least one of the warnings.